apps/useradmin/src/useradmin/metrics.py

```diff
@@ -32,6 +34,14 @@ def active_users():
         first_login=False,
         hostname__isnull=False
     ).count()
+  except OperationalError as oe:
+    connection.close()
+    connection.connect()
```

Comment:
Curious: Shouldn't we close the connection again after collecting the below metric?

Reply:
no, if the we have other db calls

Reply:
Can `connection.close()` itself throw an exception?